tests/console/yast2_http.pm

```diff
@@ -0,0 +1,89 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use strict;
+use base "consoletest";
+use testapi;
+
+
+#test yast http-server functionality -- please check test results at: http://e13.suse.de/tests/353
```

Comment:
Please remove the reference to the test. It's fine to put the test link a github comment, but not in code